src/components/LoansByCategory/HelpmeChoose/HelpmeChooseRecommendations.vue

```diff
@@ -61,16 +43,44 @@ export default {
 		HelpmeChooseBorrowerSelector,
 		KvMaterialIcon
 	},
+	props: {
+		loans: {
+			type: Array,
+			default: () => []
+		},
+	},
 	data() {
 		return {
 			mdiRefresh,
-			selectedLoan: 0
+			selectedLoan: 0,
 		};
 	},
+	computed: {
+		loanData() {
+			return this.loans[this.selectedLoan];
+		},
+		activityName() {
```

Comment:
Looks like these two computed can be removed, since you're using the methods.

Reply:
agree

Reply:
Sure, thanks!